src/parsing/Parse_rule.ml

```diff
@@ -879,7 +912,10 @@ let parse_one_rule ~rewrite_rule_ids (i : int) (rule : G.expr) : Rule.t =
   let mode_opt = take_opt rd env parse_string_wrap "mode" in
   let mode = parse_mode env mode_opt rd in
   let metadata_opt = take_opt_no_env rd (generic_to_json rule_id) "metadata" in
-  let product = parse_product rd metadata_opt in
+  let dep_formula_opt =
+    take_opt rd env parse_dependency_formula "r2c-internal-project-depends-on"
```

Comment:
When are we gonna use a proper name for this? Do customers write custom SCA rules?
Would be cleaner to have just project-depends-on:

Reply:
We don't support custom sca rules in the app, but yeah we really should have a proper name for it 😅